packages/turf-polygon-tangents/index.js

```diff
@@ -0,0 +1,84 @@
+var coordEach = require('@turf/meta').coordEach;
+var helpers = require('@turf/helpers');
+var turfPoint = helpers.point;
+var turfFc = helpers.featureCollection;
+
+/**
+ * Finds the tangents of a {@link Polygon} from a {@link Point}.
+ *
+ * @name polygonTangents
+ * @param {Feature<Point>} point to calculate the tangent points from
+ * @param {Feature<Polygon>} polygon to get tangents from
+ * @returns {FeatureCollection|Feature<Point>} Feature Collection containing the two tangent points
+ * @example
+ * var poly = {
+ *   "type": "Feature",
+ *   "properties": {},
+ *   "geometry": {
+ *     "type": "Polygon",
+ *     "coordinates": [
+ *        [
+ *          [11, 0],
+ *          [22, 4],
+ *          [31, 0],
+ *          [31, 11],
+ *          [21, 15],
+ *          [11, 11],
+ *          [11, 0]
+ *        ]
+ *      ]
+ *   }
+ * }
+ * var point = {
+ *   "type": "Feature",
+ *   "properties": {},
+ *   "geometry": {
+ *     "type": "Point",
+ *     "coordinates": [61, 5]
+ *   }
+ * }
+ * var tangents = turf.polygonTangents(point, poly)
+ * //addToMap
+ * var addToMap = [tangents];
+ */
+module.exports = function (point, polygon) {
+    var eprev;
+    var enext;
+    var rtan = 0;
+    var ltan = 0;
+
+    coordEach(polygon, function (currentCoords, currentIndex) {
+        if (currentIndex === 0) {
+            eprev = isLeft(polygon.geometry.coordinates[0][0],  polygon.geometry.coordinates[0][1], point.geometry.coordinates);
```

Comment:
Might want to consider using a shorter variable to declare your coordinates instead of always having to use `polygon.geometry.coordinates` (hard to read).

```javascript
var polyCoords = getCoords(polygon);
var ptCoords = getCoords(point);
```
Also this normalizes retrieving coordinates from either Feature or Geometry Object.